### ISSUE TITLE
'get-' implied by command line, but not respected in Get-Command

### ISSUE THREAD

**Issue by Dweeberly** [PowerShell/PowerShell#14593]
## Steps to reproduce

```
At the PowerShell command prompt enter 'uptime' or 'childitem'.  Not the command will be processed as if it were prepended by 'Get-'.  However, "get-command uptime" returns "not recognized as a name"

```

## Expected behavior

```
If the command parser recognizes and executes a command, I expect 'get-command' to show me the information about that command. 

```

## Actual behavior

```
Error message such as this "Get-Command: The term 'uptime' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again." is returned.

```

## Environment data

```Name                           Value
----                           -----
PSVersion                      7.1.0
PSEdition                      Core
GitCommitId                    7.1.0
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by mklement0** [comment]
Good find, but it has been reported before: #3987

**Comment by iSazonov** [comment]
I can not reproduce. Need more info.

**Comment by mklement0** [comment]
The default verb logic allows you to submit `ChildItem` in order to invoke `Get-ChildItem`.

However, the `Get-Command` cmdlet doesn't know about this shortcut, even though it should:

```powershell
Get-Command ChildItem # FAILS
```

Generally, relying on the default-verb logic should be avoided, both for conceptual clarity and for performance reasons, as discussed in the aforementioned duplicate, #3987.

**Comment by Dweeberly** [comment]
Sorry, clearly need to put more time into my issues search skills.  To me this is a question of completeness.  If 'uptime' (for example) produces output, then 'get-command' (or 'command') should resolve it.  I stumbled across this when I typed 'uptime' in the wrong window and then wondered why that worked.  My first response was to type get-command to figure out what just ran ... down (the shallow) rabbit hole.  I don't work with PS all the time and a lot of that is PS v5.0.  Knowing what is running and from where is rather useful.  Keeping up with all the changes to PS, .NET, C#, ... is for me a bit overwhelming since it isn't where I primarily work.  Anyway, sorry to cause work by posting a known issue.  Thank you for your reply and to everyone for the hard work they puts into these projects
